.travis.yml

```diff
@@ -43,6 +43,7 @@ matrix:
         - node ./bin/browsertime.js -b firefox test/navigationscript/multi.js https://www.sitespeed.io/blog/ -n 1
         - node ./bin/browsertime.js -b chrome test/navigationscript/multi.js -n 3 --chrome.timeline --video --visualMetrics --visualElements --viewPort 1000x600
         - node ./bin/browsertime.js -b firefox --firefox.geckoProfiler -n 1 https://www.sitespeed.io/ --spa
+        - node ./bin/browsertime.js -b firefox test/unifiedscripts/sample.js
```

Comment:
Thinking it is ok to run on one OS, since if its broken its 99,99% broken on all platforms? I haven't documented it (sorry) but the other tests on OS X and Windows is mostly to verify browsers that only works on that platform or simple generic tests.